src/Rings/mpoly.jl

```diff
@@ -311,7 +311,7 @@ function singular_generators(B::IdealGens, monorder::MonomialOrdering=default_or
   singular_assure(B)
   # in case of quotient rings, monomial ordering is ignored so far in singular_poly_ring
   isa(B.gens.Ox, MPolyQuoRing) && return B.gens.S
-  isdefined(B, :ord) && B.ord == monorder && monomial_ordering(B.Ox, ordering(base_ring(B.S))) == B.ord && return B.gens.S
+  isdefined(B, :ord) && B.ord == monorder && monomial_ordering(B.Ox, Singular.ordering(base_ring(B.S))) == B.ord && return B.gens.S
```

Comment:
@joschmitt : This change leads to a use of 30% of ~4.3s computation time in an example of @simonbrandhorst and mine. The issue seems to be that a new monomial ordering is created and then comparison can not rely on caching. Can we find a workaround for this?

Reply:
I don't see how this change can have a performance impact? The only thing happening in this line should be that `ordering` is not exported from Singular anymore. One could also do 
```suggestion
  isdefined(B, :ord) && B.ord == monorder && monomial_ordering(B.Ox, internal_ordering(base_ring(B.S))) == B.ord && return B.gens.S
```
I think. I'm fairly certain that all I did was renaming a function.

Reply:
@HechtiDerLachs just to clarify: when you say this change causes the slow down, how did you determine this, resp. check this? By reverting just that one change and then benchmarking again?

If so, knowing the example (or a minimized version) would be essential to figure out what's going on.

Reply:
I have to admit that I did not revert this particular change to see whether it solves the problem. Sorry about this premature notification, @joschmitt . Either way: There seems to be an issue with the comparison of orderings here again. 

A small example will be hard to construct. But I have one on my machine right now and I'm in my office.